.github/SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policiy
+
+The TypeScript ESLint team and community take all security vulnerabilities seriously.
+If you have a vulnerability or other security issues to disclose:
+
+* Thank you very much, please do!
+* Please send them to us by emailing `me@joshuakgoldberg.com`
```

Comment:
> `me@joshuakgoldberg.com`

Placeholder value for now, though it can work in a pinch. We should have an actual `@typescript-eslint.io` set up instead. I've started a thread internally.